plugin/completion/lib_complete.py

```diff
@@ -301,7 +304,27 @@ def _cindex_for_version(version):
         return None
 
     @staticmethod
-    def _parse_completions(complete_results):
+    def _is_valid_result(completion_result, excluded_kinds):
+        """Check if completion is valid. Remove excluded types and unaccessible members
```

Comment:
Nit: PEP 257 complains that the first sentence of the docstring does not end with a dot.